### PR TITLE
Update CHANGELOG: Add new header for v0.2.2

### PR DESCRIPTION
**Pull request by marcvanzee**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
 
+v0.2.2
+----
+ - Added Reinforcement Learning example (examples/ppo).
+ - Fix Adafactor bug that prevented factorization.
+ - Fix scan broadcast issue in functional core.
+ - Fix initialization RNGs to work with omnistaging for jitted inits.
+ - Replaces usage of 'param' kind to 'params' collection.
+ - Fix LARS optimizer for zero param initialization.
+ - Added various examples in Linen API (in directory linen_examples).
```
Comment (avital) [inline CHANGELOG.md:12]: Can you add a link to flax/linen/README for others to learn more?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,15 @@
 Changelog
 ----------
 
+v0.2.2
+----
```
Comment (avital) [inline CHANGELOG.md:5]: Did Anselm also point out that now we turn on omnistaging by default? Or am I misremembering?

Comment (marcvanzee) [reply CHANGELOG.md:10]: Added it.

File: CHANGELOG.md (hunk 1 above)
Comment (marcvanzee) [reply CHANGELOG.md]: Done.

**Review by avital — APPROVED** [review]

**Pull request merged**